accepted/2021/experimental-package-shading.md

```diff
@@ -0,0 +1,73 @@
+# Experiment: NuGet Package Shading
+
```

Comment:
Reading through the doc makes me feel this is actually a `PrivatePackage` feature similar to `PrivateAssets` feature we got today. So can we call it so i.e. "NuGet Private Packages" or "NuGet Local Packages". I like the Private better though.

Reply:
`PrivateAssets` already enables making package references private. This is about renaming private packages and their assets so they don't conflict with other copies of those assets.  I wanted to build on the existing mechanism for private references rather than defining a new one.

I don't think `PrivateAssets` is particularly user friendly, and I want to fix that in a separate, dependent spec.